core/src/main/java/com/graphhopper/storage/Graph.java

```diff
@@ -117,4 +117,6 @@
      * @return the graph extension like a TurnCostExtension
      */
     GraphExtension getExtension();
+
+    int getBaseNode(int edge, int adjNode);
```

Comment:
Maybe `getOtherNode` would be a better name (and rename `adjNode` parameter to `node`)?

Reply:
Good idea

Reply:
done